extjsdk/src/test/java/io/vantiq/extjsdk/TestUtils.java

```diff
@@ -140,6 +151,31 @@ private void checkProps() {
         assert props.getProperty(AUTH_TOKEN_PROP).contains(FAKE_TOKEN);
         assert props.getProperty(OTHER_PROP) != null;
         assert props.getProperty(OTHER_PROP).contains(FAKE_SOURCE);
+        assert props.getProperty(Utils.SEND_PING_PROPERTY_NAME) != null;
+        assert props.getProperty(Utils.SEND_PING_PROPERTY_NAME).contains(FAKE_PINGS);
+        assert props.getProperty(Utils.PORT_PROPERTY_NAME) != null;
+        assert props.getProperty(Utils.PORT_PROPERTY_NAME).contains(FAKE_PORT);
+    }
+
+    private void checkPropBeforeObtainingServer() {
+        try {
+            Boolean noUse = Utils.obtainSendPingStatus();
+            fail("We should not get here, an exception should be thrown first");
+        } catch (Exception e) {
+            // Expected to catch exception here.
+        }
+
+        try {
+            Integer noUse = Utils.obtainTCPProbePort();
+            fail("We should not get here, an exception should be thrown first");
+        } catch (Exception e) {
+            // Expected to catch exception here.
+        }
+    }
+
+    private void clearProps() {
+        Utils.clearServerConfigProperties();
+        assert Utils.serverConfigProperties == null;
```

Comment:
We probably don't really want to expose this property as public.  The test you did is _technically_ a violation of the concurrent/sync protocol